Extension/src/LanguageServer/extension.ts

```diff
@@ -866,63 +848,82 @@ function onSelectConfiguration(): void {
     }
 }
 
-function onSelectConfigurationProvider(): void {
+async function onSelectConfigurationProvider(): Promise<void> {
     onActivationEvent();
     if (!isFolderOpen()) {
         vscode.window.showInformationMessage(localize("configuration.provider.select.first", 'Open a folder first to select a configuration provider'));
     } else {
-        selectClient().then(client => client.handleConfigurationProviderSelectCommand(), rejected => {});
+        try {
+            const client: Client = await selectClient();
+            client.handleConfigurationProviderSelectCommand();
```

Comment:
I think the previous form would have throw an exception from here if encountered within handleConfigurationProviderSelectComment().  This change will hide any exception that occurs there.

Reply:
The same seems to apply to all uses of selectClient().

Reply:
And getActiveClient()